acceptance/analyzer_test.go

```diff
@@ -605,14 +608,14 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 										h.WithFlags(append(
 											dockerSocketMount,
 											"--network", registryNetwork,
-											"--env", "CNB_REGISTRY_AUTH="+fixtures.authRegAuthConfig,
+											"--env", "CNB_REGISTRY_AUTH="+fixtures.regAuthConfig,
 											"--env", "CNB_PLATFORM_API="+platformAPI,
 										)...),
 										h.WithArgs(
 											ctrPath(analyzerPath),
 											"-daemon",
-											"-cache-image", fixtures.authRegCacheImage,
-											authRegistry.RepoName("some-image"),
+											"-cache-image", fixtures.readWriteCacheImage,
+											fixtures.readWriteAppImage,
```

Comment:
This test is explicitly verifying that the credentials passed to the analyzer were handled correctly - thus I don't think we want to provide our hard-coded permissions that bypass basic auth! We probably need one more cache image that doesn't have permissions set.

Reply:
I added a couple of fixtures without using any permissions and replace it in this test case